test/TensorFlowRuntime/dataset_api.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-run-eager-swift %swift-tensorflow-test-run-extra-options
+// BUN: %target-run-eager-swift %swift-tensorflow-test-run-extra-options
```

Comment:
BUN?

Reply:
Oops, was a stale change that ended up in the PR. It should be fixed now.